requirements.txt

```diff
@@ -4,3 +4,4 @@ python-dotenv==0.10.3
 pylint==2.3.1
 picamera==1.13  
 RPi.GPIO==0.6.5
+pytest
```

Comment:
When you write a requirements file, always remember to pin a specific version. This will reduce the chance that code will work on one person's machine but not another.